lib/modules/manager/flux/__fixtures__/ociSource.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: source.toolkit.fluxcd.io/v1beta2
```

Comment:
I am not sure how to do this, since the file is needed in the `extractAllPackageFiles()` test in `extract.spec.ts`

Reply:
According to https://github.com/renovatebot/renovate/blob/03d818a1eacbc6dbb4cf6a19b19360fee021cae9/docs/development/adding-a-package-manager.md

> The extractAllPackageFiles function takes an array of filenames as input. It returns an array of filenames and dependencies.

This would not be possible with an inline fixture, but please point me in the right place.

Reply:
yes. fs should be mocked in tests like here:
https://github.com/renovatebot/renovate/blob/11a96b72a6b2564576edbc50086dd3cb28b4a390/lib/modules/manager/bundler/extract.spec.ts#L3
https://github.com/renovatebot/renovate/blob/11a96b72a6b2564576edbc50086dd3cb28b4a390/lib/modules/manager/bundler/extract.spec.ts#L7
https://github.com/renovatebot/renovate/blob/11a96b72a6b2564576edbc50086dd3cb28b4a390/lib/modules/manager/bundler/extract.spec.ts#L37

Reply:
That makes sense, but I am not sure I can retrofit this change into this PR as it is growing already, would it be better to submit a refactoring PR to address this fixture and the others after this PR is merged?

Reply:
Resolving this, please feel free to comment if what I mentioned is not ideal.